### ISSUE TITLE
How long it cost for training stark baseline using a 1080Ti?

### ISSUE THREAD

**Issue by xjtuwh** [researchmm/Stark#6]

**Comment by xjtuwh** [comment]
I use the command from readme -- python tracking/train.py --script stark_s --config baseline --save_dir . --mode single
It maybe cost more than 20 days to finished the training. Is it right?

**Comment by MasterBin-IIAU** [comment]
Hi, we don't suggest training STARK with one GPU because it will take an extremely long time. 

We suggest that you use at least two GPUs and set the batch size as large as possible for saving time. For example, on two RTX TITAN GPUs (24G Memory / GPU), you can set the batch size to about 4x larger than the original setting on 8 GPUs. In this way, you can get similar results as we provided.

Besides, you can also try training fewer epochs for saving time. For example, you can try training 250 epochs and decrease the learning rate at the 200th epoch. From my experience, the result won't drop too much.

Thanks,
Bin

**Comment by Becomebright** [comment]
How long will the training take? @MasterBin-IIAU
